community/cloud-foundation/templates/gke/examples/gke.yaml

```diff
@@ -0,0 +1,29 @@
+# Example of the GKE template usage.
```

Comment:
This example file should be called gke_zonal to void user guesswork - also to follow the same pattern as the other example files.

Reply:
File renamed in commit `b903aa9`